draft-ietf-tls-tls13.md

```diff
@@ -3980,16 +3983,20 @@ TLS-compliant application MUST implement the following TLS extensions:
   * Negotiated Groups ("supported_groups"; {{negotiated-groups}})
   * Key Share ("key_share"; {{key-share}})
   * Pre-Shared Key ("pre_shared_key"; {{pre-shared-key-extension}})
+  * Pre-Shared Key Exchange Modes ("psk_key_exchange_modes";
+    {{pre-shared-key-exchange-modes}})
```

Comment:
Though I have to ask whether you don't need to revisit this section.  It's possible not to support PSK.